Classes/UT3GoliathTurret.uc

```diff
@@ -2,7 +2,8 @@
 // UT3GoliathTurret.uc
 // I don't think the sounds will initially work...
 // GreatEmerald, 2008, 2014
-// Copyright (c) 2012, 100GPing100 (visuals + fixed sounds)
+// Copyright © 2012, 2017 Luís 'zeluisping' Guimarães <zeluis.100@gmail.com> (visuals + fixed sounds)
```

Comment:
@100GPing100 Do you actually want your email to be in the files? That can't be good for spam protection...

Reply:
Am I supposed to try to change that for GP, I did it here because that's what he did with the Paladin and assumed it would make things easier

Reply:
Let's wait for is reply.

Reply:
Well, I don't have a problem with it, but just in case, accept the pull request with that and I'll submit a pull request to remove all references to my e-mail and replace it with something else.

Reply:
OK.